hardhat/hardhat.config.ts

```diff
@@ -73,7 +73,8 @@ task('create-user-policy', 'Creates a policy for the given user')
       args.coverageAmount,
       BigNumber.from(Math.round(addDays(new Date(), -1).getTime() / 1000)),
       BigNumber.from(Math.round(addDays(new Date(), 30).getTime() / 1000)),
-      args.ipfsHash
+      args.ipfsHash,
+      'Some policy metadata'
```

Comment:
Shouldn't this be configurable? (But I guess it doesn't matter that much).

Reply:
Yes this PR accepts it as an arg: that https://github.com/api3dao/api3-dao-dashboard/pull/304

Reply:
Oh, I'll get there soon :D